### PR TITLE
Added python solution for Wormhole Sort

### PR DESCRIPTION
**Pull request by TamKengHong**
_If any of the below don't apply to this Pull Request, mark the checkbox as done._

- [ ✓] I have tested my code.
- [ ✓] I have added my solution according to the steps [here](https://usaco.guide/general/adding-solution#steps).
- [ ✓] I have followed the code conventions mentioned [here](https://usaco.guide/general/adding-solution/#code-conventions), which include the following: 
  - I understand that if it is clear that I have not attempted to follow these guidelines (ex. if I have not used tabs to indent), my PR will be closed.
  - If changes are requested, I will re-request the review after making the changes.

### CODE REVIEW

File: content/1_General/Choosing_Lang.mdx [hunk 1]
```diff
@@ -332,9 +332,54 @@ while lo != hi:
...
+It is possible to optimise this approach even further to pass all testcases. This takes around 3.7s to run.
+
+```py
+def main():
+	f = open("wormsort.in","rb")
+	n,m = map(int, f.readline().split())
+	loc = [*map(int, f.readline().split())]
+	edges = [[] for _ in range(n)]
+	weights = []
+	lo,hi = 0, m - 1
```
Comment (bqi343) [inline content/1_General/Choosing_Lang.mdx:344]: would suggest moving this line to just before `lo` and `hi` are used

File: content/1_General/Choosing_Lang.mdx [hunk 2]
```diff
@@ -332,9 +332,54 @@ while lo != hi:
...
+	weights.sort()
+
+	while lo != hi:
+		mid = (lo + hi) // 2
+		if valid(loc, weights[mid]):
+			lo = mid + 1
+		else:
+			hi = mid
+
+	open("wormsort.out","w").write(f"{-1 if lo == 0 else weights[lo-1]}\n")
```
Comment (bqi343) [inline content/1_General/Choosing_Lang.mdx:377]: can you fix this? (as mentioned in https://forum.usaco.guide/t/general-choosing-a-language-python-solution-for-usaco-silver-wormhole-sort/3699/3?u=benq)

**Review by bqi343 — CHANGES_REQUESTED** [review]

**Comment by bqi343** [comment]
btw should we remove the solution just above this one? seems like it only passes 5/10 test cases now ...

**Comment by SansPapyrus683** [comment]
i mean the point is to demonstrate that python can be slow even when it meets the target complexity, right?

**Comment by bqi343** [comment]
Yes, that's the point of the first Python solution in the module

**Comment by SansPapyrus683** [comment]
oh wait there's already 2 python solutions, yeah, just have one that doesn't pass the test cases

**Comment by TamKengHong** [comment]
Fixed all above
1. moved lo, hi to before bin search
2. changed to lo == m to output -1
3. removed the older solution

File: content/1_General/Choosing_Lang.mdx (hunk 1 above)
Comment (TamKengHong) [reply content/1_General/Choosing_Lang.mdx]: fixed

File: content/1_General/Choosing_Lang.mdx (hunk 2 above)
Comment (TamKengHong) [reply content/1_General/Choosing_Lang.mdx]: Fixed

**Review by bqi343 — APPROVED** [review]

**Pull request merged**